generators/app/templates/src/app/components/CustomButton/styles.js

```diff
@@ -0,0 +1,45 @@
+import { StyleSheet } from 'react-native';
+
+import { transparent, black, green, gray } from '../../../constants/colors';
+
+const iconSize = 20;
+
+const borderlessStyle = {
+  borderWidth: 0,
+  backgroundColor: transparent
+};
+
+export default StyleSheet.create({
```

Comment:
`export default` at the bottom

Reply:
i think that we have all export default of styles file like @ignaciosantise wrote.
In components maybe, but in style files i like this. Really there is not difference

Reply:
if we are changing the export position of Stylesheets, we should change all of them in another PR

Reply:
Agree, let's do it in another PR